src/components/Firestore/Collection.tsx

```diff
@@ -114,10 +68,10 @@ export const Collection: React.FC<Props> = ({ collection }) => {
       <Route
         path={`${url}/:id`}
         render={({ match }: any) => {
-          // TODO: what happens with missing docs?
-          const doc = state.documents.find(d => d.ref.id === match.params.id);
+          const doc = docs.find(d => d.ref.id === match.params.id);
```

Comment:
Let's use `docRef = collection.reference(match.params.id)` since we don't need snapshot anymore in `<Document>`.

Reply:
Done